src/barkhaPJun20/ExtractNumFromURL.java

```diff
@@ -0,0 +1,37 @@
+package barkhaPJun20;
+
+public class ExtractNumFromURL {
+	String output="";
+	
+	void displayItemId(String URL) {
```

Comment:
This gives correct output @BarkhaPJun20 
But there other function as well
Like index of(), substring()

This is also recommend option which write